tests/integration/jersey-2689/pom.xml

```diff
@@ -81,7 +81,7 @@
                 <artifactId>maven-failsafe-plugin</artifactId>
             </plugin>
             <plugin>
-                <groupId>org.mortbay.jetty</groupId>
```

Comment:
Why this has been updated in just these two tests? Jetty is used many places

Reply:
@jansupol yeah I agree it's good to be consistent there, just wasn't sure if making that upgrade should be done in the scope of this PR or separately :)